src/main/java/edu/wpi/grip/core/Socket.java

```diff
@@ -123,12 +124,27 @@ public boolean isPublished() {
         return this.published;
     }
 
+    /**
+     * @param connected If true, this socket is automatically set by a connection
+     */
+    public void setConnected(boolean connected) {
+        this.connected = connected;
+    }
+
+    /**
+     * @return Weather or not this socket is automatically set
```

Comment:
This could be phrased better as `True when this socket is automatically connected`